Ginger/GingerCoreCommon/WorkSpaceLib/Solution.cs

```diff
@@ -585,5 +585,8 @@ public void SetSealightsOldConifurationsToNewObject()
 
         [IsSerializedForLocalRepository]
         public AskLisaConfiguration AskLisaConfiguration { get; set; } = new AskLisaConfiguration();
+
+        [IsSerializedForLocalRepository]
```

Comment:
Please move it to new application-specific configuration xml instead of saving it on solution xml